src/index.html

```diff
@@ -21,7 +21,370 @@
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav>
+        <ul class="nav">
+          <li class="nav__list">
+            <a
+              href="#apple"
+              class="nav__link is-active"
+            >
+              apple
+            </a>
+          </li>
+
+          <li class="nav__list">
+            <a
+              href="#samsung"
+              class="nav__link"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li class="nav__list">
+            <a
+              href="#smartphones"
+              class="nav__link"
+            >
+              smartphones
+            </a>
+          </li>
+
+          <li class="nav__list">
+            <a
+              href="#laptops & Computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__list">
+            <a
+              href="#gadgets"
+              class="nav__link"
+            >
+              gadgets
+            </a>
+          </li>
+
+          <li class="nav__list">
+            <a
+              href="#tablets"
+              class="nav__link"
+            >
+              tablets
+            </a>
+          </li>
+
+          <li class="nav__list">
+            <a
+              href="#photo"
+              class="nav__link"
+            >
+              photo
+            </a>
+          </li>
+
+          <li class="nav__list">
+            <a
+              href="#video"
+              class="nav__link"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="container-catalog">
+      <div
+        class="product"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="Product Image"
+          class="product-image"
+        />
+        <div class="product-info">
+          <h2 class="product-title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="product-code">Product code: 195434</p>
+          <div class="product-star">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="product-reviews">Reviews: 5</p>
+          </div>
+          <div class="product-price">
+            <p class="product-price-text">Price:</p>
+            <span class="product-price-number">$2,199</span>
+          </div>
+          <a
+            href="#"
+            class="product-btn"
+            data-qa="hover"
+          >
+            BUY
+          </a>
+        </div>
+      </div>
+      <div class="product">
```

Comment:
Add the blank line between the cards
```suggestion
      </div>
      
      <div class="product">
```